packages/xo-web/src/xo-app/backup/health/index.js

```diff
@@ -27,6 +27,8 @@ import {
   subscribeSchedules,
 } from 'xo'
 
+import { noop } from '../../../common/utils'
```

Comment:
The path to the `common` directory is resolved.

`import { noop } from 'utils'`